### PR TITLE
Fixed function names to make them unique

### PR DESCRIPTION
**Pull request by ghost**
This fix renames non-unique functions in `intermediate-bonfires.json` to make them unique.

Overview of the changed functions:

[x] diff  -> diffArray
[x] convert -> convertToRoman
[x] where -> whereAreYou
[x] translate -> translatePigLatin
[x] pair -> pairElement
[x] boo -> booWho
[x] unite -> uniteUnique
[x] convert -> convertHTML
[x] find -> findElement
[x] drop -> dropElements
[x] steamroller -> steamrollArray
[x] every -> truthCheck
[x] add -> addTogether

Tested locally and `npm test` passes all tests.

References #7085

Edit: Fixing failed check

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/intermediate-bonfires.json [hunk 1]
```diff
@@ -811,18 +811,18 @@
         "every([{\"user\": \"Tinky-Winky\", \"sex\": \"male\"}, {\"user\": \"Dipsy\", \"sex\": \"male\"}, {\"user\": \"Laa-Laa\", \"sex\": \"female\"}, {\"user\": \"Po\", \"sex\": \"female\"}], \"sex\");"
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/intermediate-bonfires.json]: Forgot to change this `every` to `truthCheck`.

File: seed/challenges/01-front-end-development-certification/intermediate-bonfires.json [hunk 2]
```diff
@@ -730,21 +730,21 @@
         "Remember to use <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Read-Search-Ask</a> if you get stuck. Try to pair program. Write your own code."
       ],
       "challengeSeed": [
-        "function steamroller(arr) {",
+        "function steamrollArray(arr) {",
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/intermediate-bonfires.json]: @SaintPeter this challenge (Steamroller) wasn't on your list. I assume that it's because you thought the function name was fairly unique enough already. So unnecessary change in this PR?

Comment (SaintPeter) [inline seed/challenges/01-front-end-development-certification/intermediate-bonfires.json]: Correct, I didn't think it needed a new function name.  I don't know that it"s "bad" per se to change it, I just didn't think it was needed.

**Comment by erictleung** [comment]
@drk7891 looks good to me! Just make that change on line 811 I commented on above and then we can merge your PR. Thanks!

**Comment by drk7891** [comment]
@SaintPeter @erictleung Should I revert the `steamroller` change as well?

**Comment by SaintPeter** [comment]
@drk7891 No, the change is fine.  No need to revert.

**Comment by erictleung** [comment]
Looks good to me :+1:
